### ISSUE TITLE
License ambiguity

### ISSUE THREAD

**Issue by gregoa** [DrHyde/perl-modules-Scalar-Type#16]
**README** says
```
You may use, modify and distribute this code under the same terms as you
may use, modify and distribute perl itself.
```
(aka Artistic or GPL-1+)

**lib/Scalar/Type.pm** says
```
This software is free-as-in-speech software, and may be used,
distributed, and modified under the terms of either the GNU
General Public Licence version 2 or the Artistic Licence. It's
up to you which one you use. The full text of the licences can
be found in the files GPL2.txt and ARTISTIC.txt, respectively.
```

*GPL-1+ ≠ GPL2*.\ 
Please clarify :)

Cheers,
gregor, Debian Perl Group

**Comment by DrHyde** [comment]
Good spot, and thanks! I'll update it to more consistently say GPL2 or Artistic.

**Comment by gregoa** [comment]
On Sun, 29 Oct 2023 12:32:44 -0700, David Cantrell wrote:

> Good spot, and thanks! I'll update it to more consistently say GPL2 or Artistic.

Thank you! 

-- 
 .''`.  https://info.comodo.priv.at -- Debian Developer https://www.debian.org
 : :' : OpenPGP fingerprint D1E1 316E 93A7 60A8 104D  85FA BB3A 6801 8649 AA06
 `. `'  Member VIBE!AT & SPI Inc. -- Supporter Free Software Foundation Europe
   `-

**Comment by DrHyde** [comment]
This fix is in version 1.0.0, which is on its way to the CPAN now
